Python/Setup/swix/Workloads/Microsoft.VisualStudio.Workload.Python.swixproj

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="SetupProjectBefore.settings" />
+
+  <ItemGroup>
+    <Package Include="$(MSBuildProjectName).res.swr" />
+
+    <Dependency Include="Microsoft.PythonTools" />
+    <Dependency Include="Microsoft.PythonTools.Web" />
+    <Dependency Include="Microsoft.PythonTools.Samples">
+      <Type>Optional</Type>
+    </Dependency>
+    <Dependency Include="Microsoft.PythonTools.Uwp">
```

Comment:
I don't think they're hard dependencies, and I'd rather make them weak dependencies before we drag in a lot of unnecessary stuff. Starting small and working our way up is my plan here

---
In reply to: [69644898](https://github.com/Microsoft/PTVS/pull/1396#discussion_r69644898) [](ancestors = 69644898)